python/ml4ir/base/model/relevance_model.py

```diff
@@ -381,6 +404,16 @@ def save(
                 max_sequence_size=self.max_sequence_size,
             ),
         )
+
+        # Save individual layer weights
+        self.file_io.make_directory(os.path.join(model_file, "layers"), clear_dir=True)
```

Comment:
Saving weights of each layer so that we can load it later in a new model for transfer learning.

Reply:
Can you add this info as pydoc?

Reply:
Added.